tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -501,3 +502,110 @@ def test_positive_access_vmware_with_custom_profile(session, module_vmware_setti
                     if key in expected_disk_value
                 }
                 assert provided_disk_value == expected_disk_value
+
+
+@pytest.mark.tier2
+def test_positive_virt_card(
+    session, target_sat, module_vmware_settings, module_location, module_org
+):
+    """Check to see that the Virtualization card appears for an imported VM
+
+    :id: 0502d5a6-64c1-422f-a9ba-ac7c2ee7bad2
+
+    :parametrized: no
+
+    :expectedresults: Virtualization card appears in the new Host UI for the VM
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    # create entities for hostgroup
+    default_loc_id = (
+        target_sat.api.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0].id
+    )
+    target_sat.api.SmartProxy(id=1, location=[default_loc_id, module_location.id]).update()
+    domain = target_sat.api.Domain(
+        organization=[module_org.id], location=[module_location]
+    ).create()
+    subnet = target_sat.api.Subnet(
+        organization=[module_org.id], location=[module_location], domain=[domain]
+    ).create()
+    architecture = target_sat.api.Architecture().create()
+    ptable = target_sat.api.PartitionTable(
+        organization=[module_org.id], location=[module_location]
+    ).create()
+    operatingsystem = target_sat.api.OperatingSystem(
+        architecture=[architecture], ptable=[ptable]
+    ).create()
+    medium = target_sat.api.Media(
+        organization=[module_org.id], location=[module_location], operatingsystem=[operatingsystem]
+    ).create()
+    lce = (
+        target_sat.api.LifecycleEnvironment(name="Library", organization=module_org.id)
+        .search()[0]
+        .read()
+        .id
+    )
+    cv = target_sat.api.ContentView(organization=module_org).create()
```

Comment:
Did you consider any fixtures for these? Not sure what your requirements are, but it seems like these are pretty basic function/module level API fixtures.

Reply:
Yeah, I considered the generic hostgroup fixture, but it is mostly stubbed out. I'm not sure we really have a fixture setup easily available that actually creates all this stuff tied together specifically. I could make a module level one, and I considered it, but it's only used here so :shrug:

Reply:
Well if you find yourself doing the same setup then I would say make the fixture, but for now it's not necessary.

Reply:
`provisioning_hostgroup` isn't really applicable here, since it also parametrizes the test, and spins up a host. Not worth using considering the overhead and time added to the test unnecessarily.